Neos.Fusion/Classes/FusionObjects/TagImplementation.php

```diff
@@ -63,6 +63,43 @@ public function isSelfClosingTag($tagName)
         return in_array($tagName, self::$SELF_CLOSING_TAGS, true) || (boolean)$this->fusionValue('selfClosingTag');
     }
 
+
+    /**
+     * The tag content
+     *
+     * @return string
+     */
+    protected function getContent()
+    {
+        return $this->fusionValue('content');
+    }
+
+    /**
+     * The tag attributes dataStructure
+     * If anything but an iterable is returned the value is casted to string
+     *
+     * @return array|string
```

Comment:
Sorry, I didn't mean the return type hint, but the `@return` _annotation_:
```suggestion
     * @return iterable|string
```